### PR TITLE
Fusesoc and linter

### PR DESCRIPTION
**Pull request by wallento**

### CODE REVIEW

File: tb/core/tb_top_verilator.cpp [hunk 1]
```diff
@@ -69,7 +69,7 @@ void dump_memory()
 {
     errno = 0;
     std::ofstream mem_file;
-    svLogicVecVal addr = {0};
+    svLogicVecVal addr = {0, 0};
```
Comment (MikeOpenHWGroup) [inline tb/core/tb_top_verilator.cpp:72]: Why is this necessary?  Is it related to using fusesoc, or is it 'cleanup'?

File: tb/core/tb_top_verilator.cpp [hunk 2]
```diff
@@ -84,7 +84,7 @@ void dump_memory()
 
         std::cout << "finished dumping memory" << std::endl;
 
-    } catch (std::ofstream::failure e) {
+    } catch (std::ofstream::failure &e) {
```
Comment (MikeOpenHWGroup) [inline tb/core/tb_top_verilator.cpp:87]: As as comment above.

File: tb/core/tb_top_verilator.cpp (hunk 1 above)
Comment (wallento) [reply tb/core/tb_top_verilator.cpp:72]: That is a cleanup to make newer GCC happy.

File: tb/core/tb_top_verilator.cpp (hunk 2 above)
Comment (wallento) [reply tb/core/tb_top_verilator.cpp:87]: Also GCC issues a warning if you use an instance not a reference.

**Comment by wallento** [comment]
Thanks, sorry, I have closed this one, because it was an accident (before testing). Will copy the comments over to the new one (#368)

**Comment by MikeOpenHWGroup** [comment]
Never be sorry for issuing a pull-request.  We are pleased to have your contribution (even if we choose not to merge it in).
